src/versions/2.0/json.ts

```diff
@@ -213,6 +274,7 @@ export async function validateJson(
       if (typeof key === "string") {
         metadata[key] = value
       } else {
+        // is this where I need to put another check for the modifier information?
```

Comment:
I think this is the right spot for that. These checks that collect values into the local `metadata` object may have to get updated, since modifiers are part of the `METADATA_SCHEMA`. It might be useful for us to take a closer look at some of this code to make sure that we get the right approach here. Then, we can add some modifiers to the test fixtures.